OptimizelySDKCore/OptimizelySDKCore/OPTLYFeatureDecision.h

```diff
@@ -0,0 +1,46 @@
+/****************************************************************************
+ * Copyright 2016, Optimizely, Inc. and contributors                        *
```

Comment:
We need correct range of years in the Optimizely Copyright's included in our SDK files.
In case of OPTLYFeatureDecision.h, this should now be 2017-2018 , instead of 2016 .